rest/server_context_test.go

```diff
@@ -163,3 +165,197 @@ func (client *MockClient) RespondToGET(url string, response *http.Response) {
 	tripper := client.Transport.(*mockTripper)
 	tripper.getURLs[url] = response
 }
+
+func TestRemoveDatabase(t *testing.T) {
```

Comment:
This test doesn't seem to be required given that `TestAllDatabaseNames` is doing the same thing, plus a bit extra.

Reply:
Let's remove TestRemoveDatabase.